tasks/preflight.yml

```diff
@@ -107,4 +107,5 @@
       with_items: "{{ _checksums }}"
       when:
         - "('linux-' + go_arch + '.tar.gz') in item"
+  delegate_to: localhost
```

Comment:
What will happen in mixed arch environment, when `go_arch` on one host is different from `go_arch` on the other?

Reply:
Yes, it still works, same way the binary download works with `delegate_to` and `go_arch`.